ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -25,6 +25,20 @@ FAN_SS:
   implements:
   - FAN
   - SS
+  
+FAN_RMM:
+  description: "Basic fan with run mode monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - RMM  
+
+FAN_RDM:
```

Comment:
Where are the rest of the abstract types? This is not a complete fan type.

Reply:
removed the canonical....